cms/envs/aws.py

```diff
@@ -609,5 +609,4 @@
 
 derive_settings(__name__)
 
-logging.config.dictConfig(LOGGING)
```

Comment:
I think this renders the import of `logging.config` in these files redundant also.

Reply:
Ha - oh right